Adding --profile flag info (#4143)
fishtown-analytics/docs.getdbt.com: website/docs/reference/commands/init.md

- Create a new folder with your project name and sample files, enough to get you started with dbt
- Create a connection profile on your local machine. The default location is `~/.dbt/profiles.yml`. Read more in [configuring your profile](/docs/core/connect-data-platform/connection-profiles).

<VersionBlock firstVersion="1.7">

> Use the `--profile` flag with  `dbt init` to initialize the project with an existing `profiles.yml` as the `profile:` key instead of creating a new one.

Reviewer: sugg a reword to simplify. i had to reread the original a few times but may be just me. also sugg a syntax example

```suggestion
When using `dbt init` to initialize your project, include the `--profile` flag to specify an existing `profiles.yml` as the `profile:` key to use instead of creating a new one. For example, `dbt init --profile`.

```

Author: I like these changes. I think the preferred wording is "initialize" over "set up", so I'll keep that bit and roll in the remainder

> If `--profile` flag is specified with `dbt init`, and the profile exists in `profiles.yml`, the project will initialize with that as the `profile:` key instead of creating a new one.

Author: ```suggestion
Use the `--profile` flag with  `dbt init` to initialize the project with an existing `profiles.yml` as the `profile:` key instead of creating a new one.
```

> If the profile does not exist in profiles.yml or the command is run inside an existing project, the command raises an error.

Reviewer: ```suggestion
If the profile does not exist in `profiles.yml` or the command is run inside an existing project, the command raises an error.
```
